go.mod

```diff
@@ -51,3 +51,5 @@ replace (
 	github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.5
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
 )
+
+replace github.com/submariner-io/shipyard v0.9.0-rc0 => github.com/submariner-io/shipyard v0.9.0-rc0.0.20210428164906-1a52a9816269
```

Comment:
This needs to be added to the main list of dependencies (otherwise it won’t be handled during the release).